package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "omero-figure",
-  "version": "3.0.0",
+  "version": "0.0.0",
```

Comment:
A comment here that this is a placeholder and the real version in in utils.py might help avoid confusion.